cmd/mdatagen/templates/readme.md.tmpl

```diff
@@ -20,6 +20,12 @@
 {{- if ne $class "" }}
 | Issues        | ![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3A{{ $class }}%2F{{ $shortName }}%20&label=open&color=orange&logo=opentelemetry) ![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3A{{ $class }}%2F{{ $shortName }}%20&label=closed&color=blue&logo=opentelemetry) |
 {{- end }}
+{{- if .Status.Codeowners }}
+{{- $codeowners := userLinks .Status.Codeowners.Active }}
+{{- $emeritus := userLinks .Status.Codeowners.Emeritus }}
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | {{ stringsJoin $codeowners ", " }} |
+| Emeritus      | {{ stringsJoin $emeritus ", " }} |
```

Comment:
Maybe put a condition here not to generate the empty line if there are no Emeritus?

Reply:
It's not really an empty line, it shows:
```
| Emeritus |   |
```

Which indicates that there are no emeritus codeowners for this component. That in itself is a piece of information. We can also remove it entirely, in which case folks won't know we allow to record emeritus. I think I'm ok with the idea to remove that line if there are no emeritus code owners to keep the status table lean.

Reply:
I'd say it's cleaner without it, but feel free to keep it if you think otherwise

Reply:
np, fixed.